### DISCUSSION TITLE
Cyberduck S3 connection to min.IO

### DISCUSSION THREAD

**Discussion by hmiefert** [iterate-ch/cyberduck#16590] [Q&A] [answered]
Hello, I successfully established a connection to a minio server, but when I want to browse a bucket, it gives me an error "Cannot read container configuration". Oddly it tries to connect to another fqdn. I'd expected that the url would be like:
`https://<subdomain>.<domain>.<tld>/<bucketname>` but instead it tries to connect to `https://<bucketname>.<subdomain>.<domain>.<tld>`? I'm just starting with S3 as a learning topic, but to me it seems odd. Any idea, what's wrong here?

**Comment by AliveDevil** [comment, marked as answer]
In Cyberduck Preferences you can select "S3 (HTTPS, Deprecated Path Styles)" in the Profiles-tab, which will bring back the bucket access via paths.

**Comment by joubertredrat** [comment]
Hi guys,

Just another option, if you don't find "S3 (HTTPS, Deprecated Path Styles)" option, you can use "S3 (Deprecated path styles requests)" and in Server field, fill the full domain with scheme, example "https://bucket.example.com" and change port to 443.

**Comment by Mrs-Feathers** [comment]
did they remove these options? the deprecated path styles option doesntshow up anymroe and im unable to change the port from 443. could it be due to using the free version?

**Reply by hmiefert** [reply]
Go to Preferences > Profiles, wait for the list to load, then use the search field and type "deprecated".
